docs/_config.yml

```diff
@@ -6,6 +6,7 @@ color_scheme: alfresco
 url: https://alfresco.github.io
 
 aux_links:
+  Hyland Support: https://support.hyland.com/p/alfresco
   Docs: https://docs.alfresco.com/
```

Comment:
should we remove this since it's deprecated or rename it to legacy docs?

Reply:
I would rename it. 7.3 and 7.4 are still supported but not covered in new docs